presto-password-authenticators/src/main/java/com/facebook/presto/password/LdapAuthenticatorFactory.java

```diff
@@ -41,6 +45,10 @@ public PasswordAuthenticator create(Map<String, String> config)
                     binder -> {
                         configBinder(binder).bindConfig(LdapConfig.class);
                         binder.bind(LdapAuthenticator.class).in(Scopes.SINGLETON);
+
```

Comment:
It seems like an overkill to use Guice here. Just go with a PropertyManager and create a `LdapAuthenticator` manually. Make sure you check all the properties are being used. That is the functionallity that comes with a Bootstrap